src/backends/sentry_backend_breakpad.cpp

```diff
@@ -142,7 +164,9 @@ sentry__breakpad_backend_callback(
 #ifndef SENTRY_PLATFORM_WINDOWS
     sentry__leave_signal_handler();
 #endif
-    return succeeded;
+
+    // further handling can be skipped via on_crash hook
+    return already_handled || succeeded;
```

Comment:
not sure about this one. probably some tests are needed

Reply:
If I remember correctly, the breakpad docs are a bit misleading here. You should just return the `succeeded` flag unchanged. But I might be wrong, like I said, the docs are a bit unclear.

Reply:
Well "we" are not capturing the crash report, which is all that matters from our perspective. I would just pass on the `succeeded` parameter unchanged.